node/p2p_block_sync.md

```diff
@@ -0,0 +1,81 @@
+# P2P Block Sync
+
+## Abstract
+
+P2P Block Sync enables rollkit full nodes including aggregators to gossip blocks amongst  
+themselves and sync with the rollup chain faster than they can sync using the DA layer.
```

Comment:
While the implication is that it is faster, from a spec writing perspective, should we describe it in terms of roll up specific block times?

Reply:
Great point, updated it.